src/Address.svelte

```diff
@@ -58,16 +58,21 @@
   }
 </style>
 
-<div class="address" class:no-badge={noBadge}>
-  <span class="icon"><Blockies address={address} /></span>
+{#if loading}
+  <Loading fadeIn/>
+{:else}
+<div class="address" title={address} class:no-badge={noBadge}>
+  {#if !noAvatar}
+    <Avatar source={registration?.avatar ?? address} />
+  {/if}
   {#if addressType === AddressType.Org}
     <a use:link href={`/orgs/${address}`}>{addressLabel}</a>
     <span class="badge">org</span>
   {:else if addressType === AddressType.Safe}
     <a href={safeLink(address, config)} target="_blank">{addressLabel}</a>
     <span class="badge safe">safe</span>
   {:else}
```

Comment:
I'd rather explicitly check for `EOA` here in case we add another variant later.

Reply:
To avoid misunderstandings, you say to heck for `EOA` before doing the label check?
In this case we should check after or before `EOA` for `Contract` also

Reply:
In the case of a contract, we should potentially create a link to an organisation I understand, or could there be eg. multisig users?

Reply:
3dda56ee20ec6874230e28215b18dca7a3860a05 fixes this, I'm not quite sure about the final `noBadge` else if, if you have a suggestion please let me know